src/aics-image-viewer/components/ChannelsWidget.tsx

```diff
@@ -86,7 +86,7 @@ const ChannelsWidget: React.FC<ChannelsWidgetProps> = (props: ChannelsWidgetProp
     const channelName = channelNames[channelIndex];
     return channelName ? (
       <ChannelsWidgetRow
-        key={`${channelIndex}_${channelName}_${channelIndex}`}
+        key={channelIndex}
```

Comment:
what does this change do? (or, why was it so much more complicated before?)

Reply:
Makes sense.  Still, odd that channel index was in there twice, before.

I can vaguely remember considering that if the channel name is different, then there;s a reasonable chance its data range and distribution could be wildly different.  as in a user moving between two unrelated images.